systolic/CgraSystolicArrayRTL.py

```diff
@@ -82,7 +82,9 @@ def construct(s, DataType, PredicateType, CtrlPktType, CtrlSignalType,
                                  1, 1,
                                  controller2addr_map,
                                  idTo2d_map)
-    s.ctrl_ring = RingNetworkRTL(CtrlPktType, CtrlRingPos, s.num_tiles, 1)
+    # An additional port for Controller to receive CMD_COMPLETE signal from Ring to Cpu.
+    #                                                                             For latency per hop.

```

Comment:
`# For latency per hop.` -> `# The last argument of 1 is for the latency per hop`.